docs/ardana/testing.md

```diff
@@ -210,6 +210,229 @@ prepare_infra
+    that can actually be deployed on bare-metal (for example
+  [cloud-ardana8-job-entry-scale-kvm-gerrit-x86_64](https://ci.nue.suse.com/job/cloud-ardana8-job-entry-scale-kvm-gerrit-x86_64))
+    * a custom `gerrit_context` value to indicate what the new job does
+    (e.g. `entry-scale-bare-metal`).
+
+As can be seen, these use-cases are supported by allowing the user to
+specify a different Ardana integration Jenkins job to be run against
+the Gerrit change instead of the default one.
+The custom `integration_test_job` Jenkins job can either be one of the
+existing jobs that have already been provided for this purpose - those
+with their name ending in `-gerrit-x86_64` (**recommended**), or it can even
+be the fully customizable core [openstack-ardana](https://ci.nue.suse.com/job/openstack-ardana)
+job itself, in which case some or all of its parameters may need to be
+customized by using the `extra_params` parameter.
+
+E.g. to have a fully customizable `openstack-ardana` job running for a cloud9 Gerrit
+change, one might trigger a [manual cloud 9 Gerrit CI build](https://ci.nue.suse.com/job/openstack-ardana-gerrit-cloud9/build)
+with the following parameter values:
+
+  * `ardana_env` set to the IRC or LDAP username of the user (i.e. this
+  will be a private Ardana ECP deployment environment that remains running
+  and can be accessed after the job is done)
+  * `reserve_env` unchecked
+  * `voting` set, to have the new build replace the official voting CI
+  job and determine whether the Gerrit change can be merged or not
+  * `integration_test_job` set to `openstack-ardana`
+  * a custom `gerrit_context` value to indicate what the new job does
+  (e.g. `my-very-special-input-model`)
+  * the multi-line `extra_params` parameter set to the list of
+  [openstack-ardana](https://ci.nue.suse.com/job/openstack-ardana/build)
+  build parameters that are mandatory (need to be supplied) or need to be
+  overridden (different than their default values). For example, the following
+  set of `extra_params` parameters can be used to deploy a `demo` like input model
+  (integrated deployer, one controller, one SLES compute, monasca service disabled):
+
+  ```
+  scenario_name=standard
+  clm_model=integrated
+  controllers=1
+  sles_computes=1
+  disabled_services=monasca|logging|ceilometer|cassandra|kafka|spark|storm|octavia
+  ses_rgw_enabled=false
+  tempest_filter_list=ci
+  ```
+
+
+If a particular customized `openstack-ardana` scenario needs to be used
+often, it is recommended that it be added with a GitHub pull-request to
+the list of `-gerrit-x86_64` jobs predefined for
+[cloud8](https://github.com/SUSE-Cloud/automation/blob/master/jenkins/ci.suse.de/cloud-ardana8-gerrit.yaml)
+and/or [cloud9](https://github.com/SUSE-Cloud/automation/blob/master/jenkins/ci.suse.de/cloud-ardana9-gerrit.yaml),
+and thus enable other users to point to it by name.
+
+Ultimately, some Gerrit changes might not require integration testing
```

Comment:
See my comments above - I would prefer we define a special no-op job to use in this case, if we want to run with voting set... I think we shouldn't allow voting to be combined with an empty integration job TBH.

Reply:
Ok, let me set that up.

Reply:
I decided to remove this use-case completely.